src/adapters/supabase/helpers/user.ts

```diff
@@ -0,0 +1,79 @@
+import { Database } from "../types/database";
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Super } from "./supabase";
+import { Context } from "../../../types/context";
+import { addCommentToIssue } from "../../../utils/issue";
+
+const FAILED_TO_GET_USER = "Failed to get user";
+const SUCCESSFULLY_FETCHED_USER = "Successfully fetched user";
+
+export class User extends Super {
+  user_id: string | undefined;
+  comment_id: string | undefined;
+  issue_id: string | undefined;
+  repository_id: string | undefined;
+  node_id: string | undefined;
+  node_type: string | undefined;
```

Comment:
I don't believe all of these are needed

Reply:
you're right none of them are neither was one of the functions